modules/monitoring-setting-up-pod-topology-spread-constraints-for-prometheus.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-up-pod-topology-spread-constraints-for-prometheus_{context}"]
+= Setting up pod topology spread constraints for Prometheus
+
+For core {product-title} platform monitoring, you can set up pod topology spread constraints for Prometheus to fine tune how pod replicas are scheduled across zones.
+Doing so helps ensure that Prometheus pods are highly available and run more efficiently because workloads are spread across different data centers or hierarchical infrastructure zones.
+
+You configure pod topology spread constraints for core platform monitoring in the `cluster-monitoring-config` config map.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add  values for the following settings under `data/config.yaml/prometheusK8s` and `data/config.yaml/alertmanagermain` to configure topology spread constraints:
```

Comment:
If we have a dedicated section for Alertmanager
```suggestion
. Add  values for the following settings under `data/config.yaml/prometheusK8s` to configure topology spread constraints:
```

Reply:
fixed